.buildkite/build_worksheet.sh

```diff
@@ -2,46 +2,15 @@
 
 set -euo pipefail
 
-SCRIPTPATH=$(pwd)
-PIP_PATH="$SCRIPTPATH/env/bin/pip"
-PYTHON_PATH="$SCRIPTPATH/env/bin/python"
+# Goto location of this script
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-echo "Now creating virtualenv..."
-virtualenv -p python3 env
-if [ $? -ne 0 ]; then
-    echo ".. Abort!  Can't create virtualenv."
-    exit 1
-fi
+# Go to parent path (where kolibri sources are assumed to live)
+cd "$DIR"
+cd ..
+PARENT_PATH=`pwd`
+cd "$PARENT_PATH"
```

Comment:
Seems like a lot of unnecessary flip flopping around directories here. Is there any reason for this?

Reply:
This is to ensure that I can run or execute a command from the MakeFile when I'm at the `kolibri/.buildkite` path.